cli/PrintToScreen.cpp

```diff
@@ -47,6 +48,9 @@ DEFINE_bool(printing_enabled, true,
             "If true, print query results to screen normally. If false, skip "
             "printing output (e.g. for benchmarking).");
 
+int PrintToScreen::GetNumberOfDigits(std::size_t number) {
+  return number > 0 ? static_cast<int>(log10 (number)) + 1 : 1;
```

Comment:
You should use the type ``int`` for ``number``. By default ``std::size_t`` will be always positive, thus defeating the purpose of the ternary operator.